org.mixxx.Mixxx.yaml

```diff
@@ -7,22 +7,27 @@ finish-args:
   - --share=ipc
   - --socket=wayland
   - --socket=x11
+  # JACK support via PipeWire
+  - --filesystem=xdg-run/pipewire-0
+  # PulseAudio isn't actually helpful for Mixxx, but this allows access to ALSA
+  # devices too since https://github.com/flatpak/flatpak/pull/3663
   - --socket=pulseaudio
+  # Only used for MusicBrainz lookups
   - --share=network
   - --filesystem=xdg-music
   - --persist=.mixxx
   - --device=all
   - --metadata=X-DConf=migrate-path=/org/mixxx/Mixxx
+  # Required due to use of legacy GLWidget which does not work with Wayland QPA
   - --env=QT_QPA_PLATFORM=xcb
   - --env=LV2_PATH=/app/extensions/Plugins/lv2
+  # Required for battery widget
   - --system-talk-name=org.freedesktop.UPower
   - --env=ALSA_CONFIG_PATH=
-  # JACK support via pipewire
-  - --filesystem=xdg-run/pipewire-0
   - --system-talk-name=org.freedesktop.RealtimeKit1
```

Comment:
this is also part of JACK support. so it should have been moved at the same time.

Reply:
I'm not sure about that. Mixxx requests realtime priority regardless of the PortAudio backend used.

Reply:
Reference: https://github.com/flathub/org.mixxx.Mixxx/commit/03c5b46f73a55541f1ee89b674e9951b89a4a631